### PR TITLE
move gnoi generic doc to the gnoi example parent dir

### PR DESCRIPTION
**Pull request by ksator**
so we dont have the same doc in all gnoi sub directories ...

### CODE REVIEW

File: docs/examples/gnoi/gnoic.md [hunk 1]
```diff
@@ -7,33 +7,15 @@ categories:
...
 
 ## About gNOI support on EOS
 
-Please refer to this [link](https://eos.arista.com/eos-4-24-2f/gnoi/) for the gNOI support by EOS.
-
-Examples:
-
-- [gnoi ping](https://eos.arista.com/eos-4-22-1f/gnoi-ping/)
-- [gnoi traceroute](https://eos.arista.com/eos-4-22-1f/gnoi-traceroute/)
+Please refer to [this section](../index.md)
```
Comment (sulrich) [inline docs/examples/gnoi/gnoic.md:18]: might i suggest simply removing this?  the restructuring makes sense overall.

File: docs/examples/gnoi/grpcurl.md [hunk 2]
```diff
@@ -7,33 +7,15 @@ categories:
...
 
 ## About gNOI support on EOS
 
-Please refer to this [link](https://eos.arista.com/eos-4-24-2f/gnoi/) for the gNOI support by EOS.
-
-Examples:
-
-- [gnoi ping](https://eos.arista.com/eos-4-22-1f/gnoi-ping/)
-- [gnoi traceroute](https://eos.arista.com/eos-4-22-1f/gnoi-traceroute/)
+Please refer to [this section](../index.md)
```
Comment (sulrich) [inline docs/examples/gnoi/grpcurl.md:18]: ibid

**Review by sulrich — CHANGES_REQUESTED** [review]

File: docs/examples/gnoi/gnoic.md (hunk 1 above)
Comment (ksator) [reply docs/examples/gnoi/gnoic.md:18]: done. Thank you!

File: docs/examples/gnoi/grpcurl.md (hunk 2 above)
Comment (ksator) [reply docs/examples/gnoi/grpcurl.md:18]: done! Thanks!

**Review by sulrich — APPROVED** [review]

**Pull request merged**